### PR TITLE
twis45

### PR DESCRIPTION
**Pull request by larsbergstrom**
r? @Manishearth 

Or really anybody...

[Review on Reviewable](https://reviewable.io/reviews/servo/blog.servo.org/50)

### CODE REVIEW

File: _posts/2015-12-28-twis-45.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+In the [last week](https://github.com/pulls?page=1&q=is%3Apr+is%3Amerged+closed%3A2015-12-21..2015-12-28+user%3Aservo), we landed 25 PRs in the Servo organization's repositories. Given the Christmas holiday, it's not too surprising that the number is a bit lower than usual!
+
+This week brings us some of the first progress on Stylo, which is an effort to make it possible to use Servo's style system within Firefox/Gecko.
+
+
+###Notable Additions
+
+- ecoal [added](https://github.com/servo/saltfs/pull/182) support for running Servo on Linux over EGL instead of GLX
```
Comment (jdm) [inline _posts/2015-12-28-twis-45.md]: This is misleading; the code used for WebGL can now run over EGL, but not the rest of Servo yet.

Comment (larsbergstrom) [inline _posts/2015-12-28-twis-45.md]: I assume you mean that the https://github.com/servo/rust-layers/pull/220 work hasn't landed and that this commit was only for WebGL stuff?

**Comment by larsbergstrom** [comment]
Updated!

**Comment by jdm** [comment]
LGTM.

**Pull request merged**